ansible/files/admin_api_scripts/pg_upgrade_scripts/complete.sh

```diff
@@ -152,6 +152,43 @@ EOF
         run_sql -c "update pg_extension set extowner = 'postgres'::regrole where extname = 'pgmq';"
     fi
 
+    # Patch to handle upgrading to pgsodium-less Vault
+    REENCRYPT_VAULT_SECRETS_QUERY=$(cat <<EOF
+    DO \$\$
+    BEGIN
+      IF EXISTS (SELECT FROM pg_available_extension_versions WHERE name = 'supabase_vault' AND version = '0.3.0')
```

Comment:
Is this supposed to be 0.3.0?? The PR is bumping it from 0.2.9 to 0.3.1, where does 0.3.0 come in from?

Also it feels like this patch would need to be executed even if in the future we were on e.g. 0.3.2, and we're likely going to forget to do so?

Reply:
Yep, the query runs for all versions >= 0.3.0. Vault 0.3.1 would still have 0.3.0 in `pg_available_extension_versions`